src/helpers/queries.js

```diff
@@ -38,8 +38,8 @@ export const getPapersByLabel = label => ({
         type: 'file',
         trashed: false
       })
-      .indexFields(['created_at', 'metadata.qualification'])
-      .sortBy([{ created_at: 'desc' }]),
+      .indexFields(['name', 'metadata.qualification'])
+      .sortBy([{ name: 'asc' }]),
```

Comment:
Note this query won't work on PouchDB: https://github.com/pouchdb/pouchdb/issues/8385

Reply:
Ok, merci pour l'info 👍 
Peut-on quand même merger ? J'aurais besoin de cette modification très rapidement